{"level":"ERROR","location":"[get_call:164] fyersModel","message":{"API":"/history","error":"HTTPSConnectionPool(host='api-t1.fyers.in', port=443): Max retries exceeded with url: /data/history?symbol=NSE%3ANIFTY2591624750CE&resolution=1&date_format=1&range_from=2025-08-24&range_to=2025-09-10&cont_flag=1 (Caused by NameResolutionError(\"<urllib3.connection.HTTPSConnection object at 0x00000237FB584E90>: Failed to resolve 'api-t1.fyers.in' ([Errno 11001] getaddrinfo failed)\"))"},"timestamp":"2025-09-10 13:53:13,268+0530","service":"FyersAPI"}
{"level":"ERROR","location":"[get_call:164] fyersModel","message":{"API":"/history","error":"HTTPSConnectionPool(host='api-t1.fyers.in', port=443): Max retries exceeded with url: /data/history?symbol=NSE%3ANIFTY2591624750CE&resolution=1&date_format=1&range_from=2025-08-24&range_to=2025-09-10&cont_flag=1 (Caused by NameResolutionError(\"<urllib3.connection.HTTPSConnection object at 0x00000237FC92FAD0>: Failed to resolve 'api-t1.fyers.in' ([Errno 11001] getaddrinfo failed)\"))"},"timestamp":"2025-09-10 13:53:15,279+0530","service":"FyersAPI"}
{"level":"ERROR","location":"[get_call:164] fyersModel","message":{"API":"/history","error":"HTTPSConnectionPool(host='api-t1.fyers.in', port=443): Max retries exceeded with url: /data/history?symbol=NSE%3ANIFTY2591624750CE&resolution=1&date_format=1&range_from=2025-08-24&range_to=2025-09-10&cont_flag=1 (Caused by NameResolutionError(\"<urllib3.connection.HTTPSConnection object at 0x00000237FC90B0B0>: Failed to resolve 'api-t1.fyers.in' ([Errno 11001] getaddrinfo failed)\"))"},"timestamp":"2025-09-10 13:53:17,284+0530","service":"FyersAPI"}
{"level":"ERROR","location":"[get_call:164] fyersModel","message":{"API":"/history","error":"HTTPSConnectionPool(host='api-t1.fyers.in', port=443): Max retries exceeded with url: /data/history?symbol=NSE%3ANIFTY2591624750CE&resolution=1&date_format=1&range_from=2025-08-24&range_to=2025-09-10&cont_flag=1 (Caused by NameResolutionError(\"<urllib3.connection.HTTPSConnection object at 0x00000237FC9098B0>: Failed to resolve 'api-t1.fyers.in' ([Errno 11001] getaddrinfo failed)\"))"},"timestamp":"2025-09-10 13:53:19,288+0530","service":"FyersAPI"}
{"level":"ERROR","location":"[get_call:164] fyersModel","message":{"API":"/history","error":"HTTPSConnectionPool(host='api-t1.fyers.in', port=443): Read timed out. (read timeout=None)"},"timestamp":"2025-09-10 14:43:43,487+0530","service":"FyersAPI"}
{"level":"ERROR","location":"[get_call:164] fyersModel","message":{"API":"/history","error":"HTTPSConnectionPool(host='api-t1.fyers.in', port=443): Max retries exceeded with url: /data/history?symbol=NSE%3ANIFTY2591624750CE&resolution=1&date_format=1&range_from=2025-08-24&range_to=2025-09-10&cont_flag=1 (Caused by NameResolutionError(\"<urllib3.connection.HTTPSConnection object at 0x00000237FC909DC0>: Failed to resolve 'api-t1.fyers.in' ([Errno 11001] getaddrinfo failed)\"))"},"timestamp":"2025-09-10 14:43:45,497+0530","service":"FyersAPI"}
{"level":"ERROR","location":"[get_call:164] fyersModel","message":{"API":"/history","error":"HTTPSConnectionPool(host='api-t1.fyers.in', port=443): Max retries exceeded with url: /data/history?symbol=NSE%3ANIFTY2591624750CE&resolution=1&date_format=1&range_from=2025-08-24&range_to=2025-09-10&cont_flag=1 (Caused by NameResolutionError(\"<urllib3.connection.HTTPSConnection object at 0x00000237FC90A7B0>: Failed to resolve 'api-t1.fyers.in' ([Errno 11001] getaddrinfo failed)\"))"},"timestamp":"2025-09-10 14:43:47,508+0530","service":"FyersAPI"}
{"level":"ERROR","location":"[get_call:164] fyersModel","message":{"API":"/history","error":"HTTPSConnectionPool(host='api-t1.fyers.in', port=443): Max retries exceeded with url: /data/history?symbol=NSE%3ANIFTY2591624750CE&resolution=1&date_format=1&range_from=2025-08-24&range_to=2025-09-10&cont_flag=1 (Caused by NameResolutionError(\"<urllib3.connection.HTTPSConnection object at 0x00000237FC90A270>: Failed to resolve 'api-t1.fyers.in' ([Errno 11001] getaddrinfo failed)\"))"},"timestamp":"2025-09-10 14:43:49,514+0530","service":"FyersAPI"}
{"level":"ERROR","location":"[get_call:164] fyersModel","message":{"API":"/history","error":"HTTPSConnectionPool(host='api-t1.fyers.in', port=443): Max retries exceeded with url: /data/history?symbol=NSE%3ANIFTY2591624750CE&resolution=1&date_format=1&range_from=2025-08-24&range_to=2025-09-10&cont_flag=1 (Caused by NameResolutionError(\"<urllib3.connection.HTTPSConnection object at 0x00000237FC909670>: Failed to resolve 'api-t1.fyers.in' ([Errno 11001] getaddrinfo failed)\"))"},"timestamp":"2025-09-10 14:43:51,519+0530","service":"FyersAPI"}
{"level":"ERROR","location":"[get_call:164] fyersModel","message":{"API":"/history","error":"HTTPSConnectionPool(host='api-t1.fyers.in', port=443): Max retries exceeded with url: /data/history?symbol=NSE%3ANIFTY2591624750CE&resolution=1&date_format=1&range_from=2025-08-24&range_to=2025-09-10&cont_flag=1 (Caused by NameResolutionError(\"<urllib3.connection.HTTPSConnection object at 0x00000237FC92FA40>: Failed to resolve 'api-t1.fyers.in' ([Errno 11001] getaddrinfo failed)\"))"},"timestamp":"2025-09-10 14:43:53,532+0530","service":"FyersAPI"}
{"level":"ERROR","location":"[get_call:164] fyersModel","message":{"API":"/history","error":"HTTPSConnectionPool(host='api-t1.fyers.in', port=443): Max retries exceeded with url: /data/history?symbol=NSE%3ANIFTY2591624750CE&resolution=1&date_format=1&range_from=2025-08-24&range_to=2025-09-10&cont_flag=1 (Caused by NameResolutionError(\"<urllib3.connection.HTTPSConnection object at 0x00000237F7BD6570>: Failed to resolve 'api-t1.fyers.in' ([Errno 11001] getaddrinfo failed)\"))"},"timestamp":"2025-09-10 14:43:55,537+0530","service":"FyersAPI"}
{"level":"ERROR","location":"[get_call:164] fyersModel","message":{"API":"/history","error":"HTTPSConnectionPool(host='api-t1.fyers.in', port=443): Max retries exceeded with url: /data/history?symbol=NSE%3ANIFTY2591624750CE&resolution=1&date_format=1&range_from=2025-08-24&range_to=2025-09-10&cont_flag=1 (Caused by NameResolutionError(\"<urllib3.connection.HTTPSConnection object at 0x00000237FC90BE30>: Failed to resolve 'api-t1.fyers.in' ([Errno 11001] getaddrinfo failed)\"))"},"timestamp":"2025-09-10 14:43:57,549+0530","service":"FyersAPI"}
{"level":"ERROR","location":"[get_call:164] fyersModel","message":{"API":"/history","error":"HTTPSConnectionPool(host='api-t1.fyers.in', port=443): Max retries exceeded with url: /data/history?symbol=NSE%3ANIFTY2591624750CE&resolution=1&date_format=1&range_from=2025-08-24&range_to=2025-09-10&cont_flag=1 (Caused by NameResolutionError(\"<urllib3.connection.HTTPSConnection object at 0x00000237FC908830>: Failed to resolve 'api-t1.fyers.in' ([Errno 11001] getaddrinfo failed)\"))"},"timestamp":"2025-09-10 14:43:59,559+0530","service":"FyersAPI"}
{"level":"ERROR","location":"[get_call:164] fyersModel","message":{"API":"/history","error":"HTTPSConnectionPool(host='api-t1.fyers.in', port=443): Max retries exceeded with url: /data/history?symbol=NSE%3ANIFTY2591624750CE&resolution=1&date_format=1&range_from=2025-08-24&range_to=2025-09-10&cont_flag=1 (Caused by NameResolutionError(\"<urllib3.connection.HTTPSConnection object at 0x00000237FC909850>: Failed to resolve 'api-t1.fyers.in' ([Errno 11001] getaddrinfo failed)\"))"},"timestamp":"2025-09-10 14:44:01,567+0530","service":"FyersAPI"}
{"level":"ERROR","location":"[get_call:164] fyersModel","message":{"API":"/history","error":"HTTPSConnectionPool(host='api-t1.fyers.in', port=443): Max retries exceeded with url: /data/history?symbol=NSE%3ANIFTY2591624750CE&resolution=1&date_format=1&range_from=2025-08-24&range_to=2025-09-10&cont_flag=1 (Caused by NameResolutionError(\"<urllib3.connection.HTTPSConnection object at 0x00000237F7BD63F0>: Failed to resolve 'api-t1.fyers.in' ([Errno 11001] getaddrinfo failed)\"))"},"timestamp":"2025-09-10 14:44:03,578+0530","service":"FyersAPI"}
{"level":"ERROR","location":"[get_call:164] fyersModel","message":{"API":"/history","error":"HTTPSConnectionPool(host='api-t1.fyers.in', port=443): Max retries exceeded with url: /data/history?symbol=NSE%3ANIFTY2591624750CE&resolution=1&date_format=1&range_from=2025-08-24&range_to=2025-09-10&cont_flag=1 (Caused by NameResolutionError(\"<urllib3.connection.HTTPSConnection object at 0x00000237FC92F560>: Failed to resolve 'api-t1.fyers.in' ([Errno 11001] getaddrinfo failed)\"))"},"timestamp":"2025-09-10 14:44:05,584+0530","service":"FyersAPI"}
{"level":"ERROR","location":"[get_call:164] fyersModel","message":{"API":"/history","error":"HTTPSConnectionPool(host='api-t1.fyers.in', port=443): Max retries exceeded with url: /data/history?symbol=NSE%3ANIFTY2591624750CE&resolution=1&date_format=1&range_from=2025-08-24&range_to=2025-09-10&cont_flag=1 (Caused by NameResolutionError(\"<urllib3.connection.HTTPSConnection object at 0x00000237FC90BFB0>: Failed to resolve 'api-t1.fyers.in' ([Errno 11001] getaddrinfo failed)\"))"},"timestamp":"2025-09-10 14:44:07,597+0530","service":"FyersAPI"}
{"level":"ERROR","location":"[get_call:164] fyersModel","message":{"API":"/history","error":"HTTPSConnectionPool(host='api-t1.fyers.in', port=443): Max retries exceeded with url: /data/history?symbol=NSE%3ANIFTY2591624750CE&resolution=1&date_format=1&range_from=2025-08-24&range_to=2025-09-10&cont_flag=1 (Caused by NameResolutionError(\"<urllib3.connection.HTTPSConnection object at 0x00000237FC9096D0>: Failed to resolve 'api-t1.fyers.in' ([Errno 11001] getaddrinfo failed)\"))"},"timestamp":"2025-09-10 14:44:09,602+0530","service":"FyersAPI"}
{"level":"ERROR","location":"[get_call:164] fyersModel","message":{"API":"/history","error":"HTTPSConnectionPool(host='api-t1.fyers.in', port=443): Max retries exceeded with url: /data/history?symbol=NSE%3ANIFTY2591624750CE&resolution=1&date_format=1&range_from=2025-08-24&range_to=2025-09-10&cont_flag=1 (Caused by NameResolutionError(\"<urllib3.connection.HTTPSConnection object at 0x00000237FC90BB30>: Failed to resolve 'api-t1.fyers.in' ([Errno 11001] getaddrinfo failed)\"))"},"timestamp":"2025-09-10 14:44:11,616+0530","service":"FyersAPI"}
{"level":"ERROR","location":"[get_call:164] fyersModel","message":{"API":"/history","error":"HTTPSConnectionPool(host='api-t1.fyers.in', port=443): Max retries exceeded with url: /data/history?symbol=NSE%3ANIFTY2591624750CE&resolution=1&date_format=1&range_from=2025-08-24&range_to=2025-09-10&cont_flag=1 (Caused by NameResolutionError(\"<urllib3.connection.HTTPSConnection object at 0x00000237FB4AF320>: Failed to resolve 'api-t1.fyers.in' ([Errno 11001] getaddrinfo failed)\"))"},"timestamp":"2025-09-10 14:44:13,625+0530","service":"FyersAPI"}
{"level":"ERROR","location":"[get_call:164] fyersModel","message":{"API":"/history","error":"HTTPSConnectionPool(host='api-t1.fyers.in', port=443): Max retries exceeded with url: /data/history?symbol=NSE%3ANIFTY2591624750CE&resolution=1&date_format=1&range_from=2025-08-24&range_to=2025-09-10&cont_flag=1 (Caused by NameResolutionError(\"<urllib3.connection.HTTPSConnection object at 0x00000237FB5845F0>: Failed to resolve 'api-t1.fyers.in' ([Errno 11001] getaddrinfo failed)\"))"},"timestamp":"2025-09-10 14:44:15,637+0530","service":"FyersAPI"}
{"level":"ERROR","location":"[get_call:164] fyersModel","message":{"API":"/history","error":"HTTPSConnectionPool(host='api-t1.fyers.in', port=443): Max retries exceeded with url: /data/history?symbol=NSE%3ANIFTY2591624750CE&resolution=1&date_format=1&range_from=2025-08-24&range_to=2025-09-10&cont_flag=1 (Caused by NameResolutionError(\"<urllib3.connection.HTTPSConnection object at 0x00000237FB0B0E60>: Failed to resolve 'api-t1.fyers.in' ([Errno 11001] getaddrinfo failed)\"))"},"timestamp":"2025-09-10 14:44:17,647+0530","service":"FyersAPI"}
{"level":"ERROR","location":"[get_call:164] fyersModel","message":{"API":"/history","error":"HTTPSConnectionPool(host='api-t1.fyers.in', port=443): Max retries exceeded with url: /data/history?symbol=NSE%3ANIFTY2591624750CE&resolution=1&date_format=1&range_from=2025-08-24&range_to=2025-09-10&cont_flag=1 (Caused by NameResolutionError(\"<urllib3.connection.HTTPSConnection object at 0x00000237FC90B260>: Failed to resolve 'api-t1.fyers.in' ([Errno 11001] getaddrinfo failed)\"))"},"timestamp":"2025-09-10 14:44:19,659+0530","service":"FyersAPI"}
{"level":"ERROR","location":"[get_call:164] fyersModel","message":{"API":"/history","error":"HTTPSConnectionPool(host='api-t1.fyers.in', port=443): Max retries exceeded with url: /data/history?symbol=NSE%3ANIFTY2591624750CE&resolution=1&date_format=1&range_from=2025-08-24&range_to=2025-09-10&cont_flag=1 (Caused by NameResolutionError(\"<urllib3.connection.HTTPSConnection object at 0x00000237FC90B530>: Failed to resolve 'api-t1.fyers.in' ([Errno 11001] getaddrinfo failed)\"))"},"timestamp":"2025-09-10 14:44:21,669+0530","service":"FyersAPI"}
{"level":"ERROR","location":"[get_call:164] fyersModel","message":{"API":"/history","error":"HTTPSConnectionPool(host='api-t1.fyers.in', port=443): Max retries exceeded with url: /data/history?symbol=NSE%3ANIFTY2591624750CE&resolution=1&date_format=1&range_from=2025-08-24&range_to=2025-09-10&cont_flag=1 (Caused by NameResolutionError(\"<urllib3.connection.HTTPSConnection object at 0x00000237FC90AE40>: Failed to resolve 'api-t1.fyers.in' ([Errno 11001] getaddrinfo failed)\"))"},"timestamp":"2025-09-10 14:44:23,675+0530","service":"FyersAPI"}
{"level":"ERROR","location":"[get_call:164] fyersModel","message":{"API":"/history","error":"HTTPSConnectionPool(host='api-t1.fyers.in', port=443): Max retries exceeded with url: /data/history?symbol=NSE%3ANIFTY2591624750CE&resolution=1&date_format=1&range_from=2025-08-24&range_to=2025-09-10&cont_flag=1 (Caused by NameResolutionError(\"<urllib3.connection.HTTPSConnection object at 0x00000237FB584290>: Failed to resolve 'api-t1.fyers.in' ([Errno 11001] getaddrinfo failed)\"))"},"timestamp":"2025-09-10 14:44:25,684+0530","service":"FyersAPI"}
{"level":"ERROR","location":"[get_call:164] fyersModel","message":{"API":"/history","error":"HTTPSConnectionPool(host='api-t1.fyers.in', port=443): Max retries exceeded with url: /data/history?symbol=NSE%3ANIFTY2591624750CE&resolution=1&date_format=1&range_from=2025-08-24&range_to=2025-09-10&cont_flag=1 (Caused by NameResolutionError(\"<urllib3.connection.HTTPSConnection object at 0x00000237FC92FAD0>: Failed to resolve 'api-t1.fyers.in' ([Errno 11001] getaddrinfo failed)\"))"},"timestamp":"2025-09-10 14:44:27,691+0530","service":"FyersAPI"}
{"level":"ERROR","location":"[get_call:164] fyersModel","message":{"API":"/history","error":"HTTPSConnectionPool(host='api-t1.fyers.in', port=443): Max retries exceeded with url: /data/history?symbol=NSE%3ANIFTY2591624750CE&resolution=1&date_format=1&range_from=2025-08-24&range_to=2025-09-10&cont_flag=1 (Caused by NameResolutionError(\"<urllib3.connection.HTTPSConnection object at 0x00000237FC90A180>: Failed to resolve 'api-t1.fyers.in' ([Errno 11001] getaddrinfo failed)\"))"},"timestamp":"2025-09-10 14:44:29,702+0530","service":"FyersAPI"}
{"level":"ERROR","location":"[get_call:164] fyersModel","message":{"API":"/history","error":"HTTPSConnectionPool(host='api-t1.fyers.in', port=443): Max retries exceeded with url: /data/history?symbol=NSE%3ANIFTY2591624750CE&resolution=1&date_format=1&range_from=2025-08-24&range_to=2025-09-10&cont_flag=1 (Caused by NameResolutionError(\"<urllib3.connection.HTTPSConnection object at 0x00000237FC90A060>: Failed to resolve 'api-t1.fyers.in' ([Errno 11001] getaddrinfo failed)\"))"},"timestamp":"2025-09-10 14:44:31,709+0530","service":"FyersAPI"}
{"level":"ERROR","location":"[get_call:164] fyersModel","message":{"API":"/history","error":"HTTPSConnectionPool(host='api-t1.fyers.in', port=443): Max retries exceeded with url: /data/history?symbol=NSE%3ANIFTY2591624750CE&resolution=1&date_format=1&range_from=2025-08-24&range_to=2025-09-10&cont_flag=1 (Caused by NameResolutionError(\"<urllib3.connection.HTTPSConnection object at 0x00000237FC90BEC0>: Failed to resolve 'api-t1.fyers.in' ([Errno 11001] getaddrinfo failed)\"))"},"timestamp":"2025-09-10 14:44:33,716+0530","service":"FyersAPI"}
{"level":"ERROR","location":"[get_call:164] fyersModel","message":{"API":"/history","error":"HTTPSConnectionPool(host='api-t1.fyers.in', port=443): Max retries exceeded with url: /data/history?symbol=NSE%3ANIFTY2591624750CE&resolution=1&date_format=1&range_from=2025-08-24&range_to=2025-09-10&cont_flag=1 (Caused by NameResolutionError(\"<urllib3.connection.HTTPSConnection object at 0x00000237FC92F110>: Failed to resolve 'api-t1.fyers.in' ([Errno 11001] getaddrinfo failed)\"))"},"timestamp":"2025-09-10 14:44:35,728+0530","service":"FyersAPI"}
{"level":"ERROR","location":"[get_call:164] fyersModel","message":{"API":"/history","error":"HTTPSConnectionPool(host='api-t1.fyers.in', port=443): Max retries exceeded with url: /data/history?symbol=NSE%3ANIFTY2591624750CE&resolution=1&date_format=1&range_from=2025-08-24&range_to=2025-09-10&cont_flag=1 (Caused by NameResolutionError(\"<urllib3.connection.HTTPSConnection object at 0x00000237FB587410>: Failed to resolve 'api-t1.fyers.in' ([Errno 11001] getaddrinfo failed)\"))"},"timestamp":"2025-09-10 14:44:37,733+0530","service":"FyersAPI"}
{"level":"ERROR","location":"[get_call:164] fyersModel","message":{"API":"/history","error":"HTTPSConnectionPool(host='api-t1.fyers.in', port=443): Max retries exceeded with url: /data/history?symbol=NSE%3ANIFTY2591624750CE&resolution=1&date_format=1&range_from=2025-08-24&range_to=2025-09-10&cont_flag=1 (Caused by NameResolutionError(\"<urllib3.connection.HTTPSConnection object at 0x00000237FC90BFE0>: Failed to resolve 'api-t1.fyers.in' ([Errno 11001] getaddrinfo failed)\"))"},"timestamp":"2025-09-10 14:44:39,742+0530","service":"FyersAPI"}
{"level":"ERROR","location":"[get_call:164] fyersModel","message":{"API":"/history","error":"HTTPSConnectionPool(host='api-t1.fyers.in', port=443): Max retries exceeded with url: /data/history?symbol=NSE%3ANIFTY2591624750CE&resolution=1&date_format=1&range_from=2025-08-24&range_to=2025-09-10&cont_flag=1 (Caused by NameResolutionError(\"<urllib3.connection.HTTPSConnection object at 0x00000237FC90A330>: Failed to resolve 'api-t1.fyers.in' ([Errno 11001] getaddrinfo failed)\"))"},"timestamp":"2025-09-10 14:44:41,748+0530","service":"FyersAPI"}
{"level":"ERROR","location":"[get_call:164] fyersModel","message":{"API":"/history","error":"HTTPSConnectionPool(host='api-t1.fyers.in', port=443): Max retries exceeded with url: /data/history?symbol=NSE%3ANIFTY2591624750CE&resolution=1&date_format=1&range_from=2025-08-24&range_to=2025-09-10&cont_flag=1 (Caused by NameResolutionError(\"<urllib3.connection.HTTPSConnection object at 0x00000237FC9085C0>: Failed to resolve 'api-t1.fyers.in' ([Errno 11001] getaddrinfo failed)\"))"},"timestamp":"2025-09-10 14:44:43,753+0530","service":"FyersAPI"}
{"level":"ERROR","location":"[get_call:164] fyersModel","message":{"API":"/history","error":"HTTPSConnectionPool(host='api-t1.fyers.in', port=443): Max retries exceeded with url: /data/history?symbol=NSE%3ANIFTY2591624750CE&resolution=1&date_format=1&range_from=2025-08-24&range_to=2025-09-10&cont_flag=1 (Caused by NameResolutionError(\"<urllib3.connection.HTTPSConnection object at 0x00000237F7BD6B40>: Failed to resolve 'api-t1.fyers.in' ([Errno 11001] getaddrinfo failed)\"))"},"timestamp":"2025-09-10 14:44:45,766+0530","service":"FyersAPI"}
{"level":"ERROR","location":"[get_call:164] fyersModel","message":{"API":"/history","error":"HTTPSConnectionPool(host='api-t1.fyers.in', port=443): Max retries exceeded with url: /data/history?symbol=NSE%3ANIFTY2591624750CE&resolution=1&date_format=1&range_from=2025-08-24&range_to=2025-09-10&cont_flag=1 (Caused by NameResolutionError(\"<urllib3.connection.HTTPSConnection object at 0x00000237FB584B90>: Failed to resolve 'api-t1.fyers.in' ([Errno 11001] getaddrinfo failed)\"))"},"timestamp":"2025-09-10 14:44:47,773+0530","service":"FyersAPI"}
{"level":"ERROR","location":"[get_call:164] fyersModel","message":{"API":"/history","error":"HTTPSConnectionPool(host='api-t1.fyers.in', port=443): Max retries exceeded with url: /data/history?symbol=NSE%3ANIFTY2591624750CE&resolution=1&date_format=1&range_from=2025-08-24&range_to=2025-09-10&cont_flag=1 (Caused by NameResolutionError(\"<urllib3.connection.HTTPSConnection object at 0x00000237FC908A40>: Failed to resolve 'api-t1.fyers.in' ([Errno 11001] getaddrinfo failed)\"))"},"timestamp":"2025-09-10 14:44:49,778+0530","service":"FyersAPI"}
{"level":"ERROR","location":"[get_call:164] fyersModel","message":{"API":"/history","error":"HTTPSConnectionPool(host='api-t1.fyers.in', port=443): Max retries exceeded with url: /data/history?symbol=NSE%3ANIFTY2591624750CE&resolution=1&date_format=1&range_from=2025-08-24&range_to=2025-09-10&cont_flag=1 (Caused by NameResolutionError(\"<urllib3.connection.HTTPSConnection object at 0x00000237FC90B860>: Failed to resolve 'api-t1.fyers.in' ([Errno 11001] getaddrinfo failed)\"))"},"timestamp":"2025-09-10 14:44:51,788+0530","service":"FyersAPI"}
{"level":"ERROR","location":"[get_call:164] fyersModel","message":{"API":"/history","error":"HTTPSConnectionPool(host='api-t1.fyers.in', port=443): Max retries exceeded with url: /data/history?symbol=NSE%3ANIFTY2591624750CE&resolution=1&date_format=1&range_from=2025-08-24&range_to=2025-09-10&cont_flag=1 (Caused by NameResolutionError(\"<urllib3.connection.HTTPSConnection object at 0x00000237FC92FE90>: Failed to resolve 'api-t1.fyers.in' ([Errno 11001] getaddrinfo failed)\"))"},"timestamp":"2025-09-10 14:44:53,796+0530","service":"FyersAPI"}
{"level":"ERROR","location":"[get_call:164] fyersModel","message":{"API":"/history","error":"HTTPSConnectionPool(host='api-t1.fyers.in', port=443): Max retries exceeded with url: /data/history?symbol=NSE%3ANIFTY2591624750CE&resolution=1&date_format=1&range_from=2025-08-24&range_to=2025-09-10&cont_flag=1 (Caused by NameResolutionError(\"<urllib3.connection.HTTPSConnection object at 0x00000237FC978470>: Failed to resolve 'api-t1.fyers.in' ([Errno 11001] getaddrinfo failed)\"))"},"timestamp":"2025-09-10 14:44:55,803+0530","service":"FyersAPI"}
{"level":"ERROR","location":"[get_call:164] fyersModel","message":{"API":"/history","error":"HTTPSConnectionPool(host='api-t1.fyers.in', port=443): Max retries exceeded with url: /data/history?symbol=NSE%3ANIFTY2591624750CE&resolution=1&date_format=1&range_from=2025-08-24&range_to=2025-09-10&cont_flag=1 (Caused by NameResolutionError(\"<urllib3.connection.HTTPSConnection object at 0x00000237FB584080>: Failed to resolve 'api-t1.fyers.in' ([Errno 11001] getaddrinfo failed)\"))"},"timestamp":"2025-09-10 14:44:57,815+0530","service":"FyersAPI"}
{"level":"ERROR","location":"[get_call:164] fyersModel","message":{"API":"/history","error":"HTTPSConnectionPool(host='api-t1.fyers.in', port=443): Max retries exceeded with url: /data/history?symbol=NSE%3ANIFTY2591624750CE&resolution=1&date_format=1&range_from=2025-08-24&range_to=2025-09-10&cont_flag=1 (Caused by NameResolutionError(\"<urllib3.connection.HTTPSConnection object at 0x00000237FC909430>: Failed to resolve 'api-t1.fyers.in' ([Errno 11001] getaddrinfo failed)\"))"},"timestamp":"2025-09-10 14:44:59,829+0530","service":"FyersAPI"}
{"level":"ERROR","location":"[get_call:164] fyersModel","message":{"API":"/history","error":"HTTPSConnectionPool(host='api-t1.fyers.in', port=443): Max retries exceeded with url: /data/history?symbol=NSE%3ANIFTY2591624750CE&resolution=1&date_format=1&range_from=2025-08-24&range_to=2025-09-10&cont_flag=1 (Caused by NameResolutionError(\"<urllib3.connection.HTTPSConnection object at 0x00000237FC9087D0>: Failed to resolve 'api-t1.fyers.in' ([Errno 11001] getaddrinfo failed)\"))"},"timestamp":"2025-09-10 14:45:01,833+0530","service":"FyersAPI"}
{"level":"ERROR","location":"[get_call:146] fyersModel","message":{"API":"/history","Error":{"code":-50,"data":{"resolution":"resolution should be 5S, 10S, 15S, 30S, 45S, 1, 2, 3, 5, 10, 15, 20, 30, 45, 60, 120, 180, 240 or 1D"},"message":"Invalid input","s":"error"}},"timestamp":"2025-09-10 15:23:17,615+0530","service":"FyersAPI"}
{"level":"ERROR","location":"[get_call:146] fyersModel","message":{"API":"/history","Error":{"code":-50,"data":{"resolution":"resolution should be 5S, 10S, 15S, 30S, 45S, 1, 2, 3, 5, 10, 15, 20, 30, 45, 60, 120, 180, 240 or 1D"},"message":"Invalid input","s":"error"}},"timestamp":"2025-09-10 15:23:19,838+0530","service":"FyersAPI"}
{"level":"ERROR","location":"[get_call:146] fyersModel","message":{"API":"/history","Error":{"code":-50,"data":{"resolution":"resolution should be 5S, 10S, 15S, 30S, 45S, 1, 2, 3, 5, 10, 15, 20, 30, 45, 60, 120, 180, 240 or 1D"},"message":"Invalid input","s":"error"}},"timestamp":"2025-09-10 15:23:22,054+0530","service":"FyersAPI"}
{"level":"ERROR","location":"[get_call:146] fyersModel","message":{"API":"/history","Error":{"code":-50,"data":{"resolution":"resolution should be 5S, 10S, 15S, 30S, 45S, 1, 2, 3, 5, 10, 15, 20, 30, 45, 60, 120, 180, 240 or 1D"},"message":"Invalid input","s":"error"}},"timestamp":"2025-09-10 15:23:24,287+0530","service":"FyersAPI"}
{"level":"ERROR","location":"[get_call:146] fyersModel","message":{"API":"/history","Error":{"code":-50,"data":{"resolution":"resolution should be 5S, 10S, 15S, 30S, 45S, 1, 2, 3, 5, 10, 15, 20, 30, 45, 60, 120, 180, 240 or 1D"},"message":"Invalid input","s":"error"}},"timestamp":"2025-09-10 15:23:26,492+0530","service":"FyersAPI"}
{"level":"ERROR","location":"[get_call:146] fyersModel","message":{"API":"/history","Error":{"code":-50,"data":{"resolution":"resolution should be 5S, 10S, 15S, 30S, 45S, 1, 2, 3, 5, 10, 15, 20, 30, 45, 60, 120, 180, 240 or 1D"},"message":"Invalid input","s":"error"}},"timestamp":"2025-09-10 15:23:28,707+0530","service":"FyersAPI"}
{"level":"ERROR","location":"[get_call:146] fyersModel","message":{"API":"/history","Error":{"code":-50,"data":{"resolution":"resolution should be 5S, 10S, 15S, 30S, 45S, 1, 2, 3, 5, 10, 15, 20, 30, 45, 60, 120, 180, 240 or 1D"},"message":"Invalid input","s":"error"}},"timestamp":"2025-09-10 15:23:30,936+0530","service":"FyersAPI"}
{"level":"ERROR","location":"[get_call:146] fyersModel","message":{"API":"/history","Error":{"code":-50,"data":{"resolution":"resolution should be 5S, 10S, 15S, 30S, 45S, 1, 2, 3, 5, 10, 15, 20, 30, 45, 60, 120, 180, 240 or 1D"},"message":"Invalid input","s":"error"}},"timestamp":"2025-09-10 15:23:33,187+0530","service":"FyersAPI"}
{"level":"ERROR","location":"[get_call:146] fyersModel","message":{"API":"/history","Error":{"code":-50,"data":{"resolution":"resolution should be 5S, 10S, 15S, 30S, 45S, 1, 2, 3, 5, 10, 15, 20, 30, 45, 60, 120, 180, 240 or 1D"},"message":"Invalid input","s":"error"}},"timestamp":"2025-09-10 15:23:35,412+0530","service":"FyersAPI"}
{"level":"ERROR","location":"[get_call:146] fyersModel","message":{"API":"/history","Error":{"code":-50,"data":{"resolution":"resolution should be 5S, 10S, 15S, 30S, 45S, 1, 2, 3, 5, 10, 15, 20, 30, 45, 60, 120, 180, 240 or 1D"},"message":"Invalid input","s":"error"}},"timestamp":"2025-09-10 15:23:37,683+0530","service":"FyersAPI"}
{"level":"ERROR","location":"[get_call:146] fyersModel","message":{"API":"/history","Error":{"code":-50,"data":{"resolution":"resolution should be 5S, 10S, 15S, 30S, 45S, 1, 2, 3, 5, 10, 15, 20, 30, 45, 60, 120, 180, 240 or 1D"},"message":"Invalid input","s":"error"}},"timestamp":"2025-09-10 15:23:39,922+0530","service":"FyersAPI"}
{"level":"ERROR","location":"[get_call:146] fyersModel","message":{"API":"/history","Error":{"code":-50,"data":{"resolution":"resolution should be 5S, 10S, 15S, 30S, 45S, 1, 2, 3, 5, 10, 15, 20, 30, 45, 60, 120, 180, 240 or 1D"},"message":"Invalid input","s":"error"}},"timestamp":"2025-09-10 15:23:42,165+0530","service":"FyersAPI"}
{"level":"ERROR","location":"[get_call:146] fyersModel","message":{"API":"/history","Error":{"code":-50,"data":{"resolution":"resolution should be 5S, 10S, 15S, 30S, 45S, 1, 2, 3, 5, 10, 15, 20, 30, 45, 60, 120, 180, 240 or 1D"},"message":"Invalid input","s":"error"}},"timestamp":"2025-09-10 15:23:44,388+0530","service":"FyersAPI"}
{"level":"ERROR","location":"[get_call:146] fyersModel","message":{"API":"/history","Error":{"code":-50,"data":{"resolution":"resolution should be 5S, 10S, 15S, 30S, 45S, 1, 2, 3, 5, 10, 15, 20, 30, 45, 60, 120, 180, 240 or 1D"},"message":"Invalid input","s":"error"}},"timestamp":"2025-09-10 15:23:46,614+0530","service":"FyersAPI"}
{"level":"ERROR","location":"[get_call:146] fyersModel","message":{"API":"/history","Error":{"code":-50,"data":{"resolution":"resolution should be 5S, 10S, 15S, 30S, 45S, 1, 2, 3, 5, 10, 15, 20, 30, 45, 60, 120, 180, 240 or 1D"},"message":"Invalid input","s":"error"}},"timestamp":"2025-09-10 15:23:48,837+0530","service":"FyersAPI"}
